sys-cluster/rdma-core/rdma-core-34.1.ebuild

```diff
@@ -83,19 +85,21 @@ src_configure() {
 		mycmakeargs+=( -DNO_PYVERBS=ON )
```

Comment:
You can use `-DNO_PYVERBS=$(usex python OFF ON)` for this, no?

Reply:
Didn't write this myself, but I think you're right. The gentoo version also sets `PYTHON_EXECUTABLE` by the way, but I'm not sure if that's really necessary (?):
https://github.com/gentoo/gentoo/blob/master/sys-cluster/rdma-core/rdma-core-34.0.ebuild#L81
Since this PR is already merged, I'll open an issue about this.

Reply:
No problem, this is just a suggestion... Some packages might need setting `PYTHON_EXECUTABLE` if their build system is fiddling with it manually, but you're right that usually it's not required. BTW, if the package is already in Gentoo, just open a PR, if it doesn't get merged you can ping me.

Reply:
I made a PR:
https://github.com/gentoo/gentoo/pull/21279